src/visitor/RewriteVisitor.h

```diff
@@ -36,7 +36,7 @@ class RewriteVisitor final : public ExprVisitorImpl {
         return rewriter_;
     }
 
-private:
+public:
```

Comment:
How about keeping it private and implementing it in a static function？[https://github.com/vesoft-inc/nebula-graph/pull/1305](https://github.com/vesoft-inc/nebula-graph/pull/1305)